### ISSUE TITLE
Lower Accuracy on Imagnet

### ISSUE THREAD

**Issue by LetheLll** [pytorch/examples#920]
I use pre-trained model in torchvision.models, but I get a lower accuracy on valid set.
```06-15 15:30 - INFO: - Test [0/196]	Time 8.538 (8.538)	Loss 0.4865(0.4865)	Prec@1 85.938(85.938)\n
06-15 15:30 - INFO: - Test [50/196]	Time 0.335 (0.497)	Loss 0.4764(0.6844)	Prec@1 89.062(82.207)
06-15 15:30 - INFO: - Test [100/196]	Time 0.335 (0.417)	Loss 1.1331(0.8115)	Prec@1 68.359(79.293)
06-15 15:30 - INFO: - Test [150/196]	Time 0.346 (0.391)	Loss 1.0446(0.9234)	Prec@1 75.781(76.922)
06-15 15:31 - INFO: - Valid error:76.012```

**Comment by mostafaelhoushi** [comment]
Which model is that?

**Comment by msaroufim** [comment]
Closing since no response from OP
